unittests/security_group_tests.cpp

```diff
@@ -421,7 +421,9 @@ BOOST_AUTO_TEST_CASE(test_participants_change_modified) {
 }
 
 BOOST_AUTO_TEST_CASE(test_participants_change_2_chains) { 
-   // chain2 trails chain a block behind since it reads the records of last finalized block from chain
+   /* Note:
+   * because produce_block calls start_block, and push_block does not, chain2 will trail behind chain
```

Comment:
Sorry, I realized that this is not clear, it need to be "because produce_block calls start_block after finalizing the produced block, and push_block does not, chain2 will trail behind chain". They both call start_block, but push_block is calling it at the start, while produce block is calling it after it finalizes the block it is producing.